todo_app/templates/index.html

```diff
@@ -2,17 +2,29 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
+<form action="/item" method="post" class="todo-form" id="todoForm">
   <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
+    <h1 class="todo-header" id="todoHeader">To-Do App</h1>
+    <p class="todo-header-para" id="todoHeaderPara">Things to-do app.</p>
   </div>
-
+  <!-- <div class="jumbotron"> -->
```

Comment:
minor, but it's always best to remove commented out code

Reply:
Thanks very much - have fixed in commit a943886